lte/cloud/go/services/subscriberdb/digest.go

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package subscriberdb
+
+import (
+	"magma/lte/cloud/go/lte"
+	"magma/lte/cloud/go/serdes"
+	lte_models "magma/lte/cloud/go/services/lte/obsidian/models"
+	"magma/orc8r/cloud/go/mproto"
+	mproto_protos "magma/orc8r/cloud/go/mproto/protos"
+	"magma/orc8r/cloud/go/services/configurator"
+
+	"github.com/golang/glog"
+	"github.com/golang/protobuf/proto"
+)
+
+const defaultSubProfile = "default"
+
+// GetDigest returns a deterministic digest of the current configurations of a
+// network, which is a concatenation of its subscribers digest and apn resources digest.
+func GetDigest(networkID string) (string, error) {
+	// HACK: Workaround to decouple apn resources data from subscriber data despite current
+	// construction logic.
+	//
+	// The subscribers and apn resources digests are separately generated so that the
+	// digests are per-network, instead of having to account for gateway-specific apn
+	// resources configs for subscribers. We can do this because apn resources are
```

Comment:
"We can do this because APN resource IDs are unique within each network" is sufficient/concise